grp/perf.gd

```diff
@@ -129,28 +129,10 @@ DeclareGlobalFunction("SizesPerfectGroups");
 ##
 DeclareGlobalFunction("NumberPerfectGroups");
 DeclareSynonym("NrPerfectGroups",NumberPerfectGroups);
+DeclareSynonym("NumberPerfectLibraryGroups",NumberPerfectGroups);
+DeclareSynonym("NrPerfectLibraryGroups",NumberPerfectGroups);
 
 
-#############################################################################
-##
-#F  NumberPerfectLibraryGroups( <size> )  . . . . . . . . . . . . . . . . . .
-##
-##  <#GAPDoc Label="NumberPerfectLibraryGroups">
```

Comment:
Note that the reference manual still references these, so you need to update `doc/ref/grplib.xml` if you remove this section.

However, instead of outright dropping these from the manual, I think it would be nice if we kept at least a minimal reference to them, something like this:
> `NumberPerfectLibraryGroups` and `NrPerfectLibraryGroups` are synonyms for `NumberPerfectGroups` which only exist for historical reasons.
That could be part of the `NumberPerfectGroups` manual entry, and we could add suitable GAPDoc commands to ensure `NrPerfectLibraryGroups` and `NumberPerfectLibraryGroups` are still in the index and point to that manual entry.

Reply:
Thank you -- yes, I will do so (and add other documentation)